wallet/src/Pos/Wallet/Web/Sockets/Notifier.hs

```diff
@@ -42,7 +42,7 @@ launchNotifier
     => (forall x. m x -> Handler x)
     -> m ()
 launchNotifier nat =
-    void . liftIO $ mapM startForking
+    void . liftIO $ mapConcurrently startNotifier
```

Comment:
`launchNotifier` becomes blocking (forever). Since it's used in wallet startup sequence as is (https://goo.gl/nbnnW9), wallet will never actually start

Reply:
Oh, yes, of course. I should make it another plugin as I did with `startPendingTxsResubmitter`. Good catch.

Reply:
My understnading is that Daedalus UI is not using notifier - so it is just a part in our code nobody is using.

My understanding is that UI is using http for all request nowdays.

Reply:
@Martoon-00 @gromakovsky see here https://input-output-rnd.slack.com/archives/C7PJ29FME/p1516873894000474?thread_ts=1516873660.000197&cid=C7PJ29FME . I believe notifier can be removed alltogether

Reply:
Created https://iohk.myjetbrains.com/youtrack/issue/CSL-2218, maybe someone will do it.